rules/review-videos-collaboratively/rule.md

```diff
@@ -28,9 +27,13 @@ So you have just finished that sweet video edit and you are now ready & excited
 
 This is where collaborative viewing platforms like [Frame.io](https://frame.io) come in. 
 
+'youtube: https://www.youtube.com/embed/rapbhpW3jPM'
+**Video What is Frame.io (2:16 min)**
```

Comment:
@williamgreentreessw 

1. Please follow https://ssw.com.au/rules/right-format-to-write-videos-time-length/ here

Reply:
Fixed markdown